ui/src/shared/components/Gauge.tsx

```diff
@@ -340,7 +351,7 @@ class Gauge extends Component<Props> {
     const textContent = formatStatValue(gaugePosition, {
       decimalPlaces,
       prefix,
-      suffix,
+      suffix
```

Comment:
did prettier do this? I think we have hanging commas set in our linter preferences

Reply:
I did not do this, so possibly we should change our prettier?

Reply:
Nah, I wouldn't worry about it